src/CBT.DotNetFx/packall.cmd

```diff
@@ -10,7 +10,7 @@ IF "%ProgramFiles(x86)%" NEQ "" (
 
 CALL :Net35
 
-FOR %%A IN (4.0.0,4.5.0,4.5.1,4.5.2,4.6.0,4.6.1,4.6.2) DO (
+FOR %%A IN (4.0.0,4.5.0,4.5.1,4.5.2,4.6.0,4.6.1,4.6.2,4.7.1,4.7.2) DO (
```

Comment:
Should there be an entry for 4.7.0 along with Before.Microsoft.Common-net47.targets?

Reply:
Maybe. I'm not 100% sure any of this is even used anyway, but am keeping it in sync regardless.

The problem is that when I look under my `C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework`, I don't see v4.6.0, I see v4.6. So even this existing value of 4.6.0 seems incorrect, which furthers my belief of this just being unused and broken.

I'll dig a little deeper and either fix this or kill it.